lib/trap.rb

```diff
@@ -32,8 +32,8 @@ def self.normal_trap(signal)
         perform_trap(signal)
       else
         @sig_timestamps[signal] = time
-        write_log("SIG#{signal}(1) received, aborting if another SIG#{signal} is"\
-                            ' received in the span of the next one second')
+        write_log("SIG#{signal}(1) received, aborting if another SIG#{signal} is " \
```

Comment:
In general, having a line break in a string literal is not great. It makes difficult to search the message with grep. Such a long message is often bad not only in the source code but actual output. It is usually better to add a line break also in the actual output or shorten the message.